src/scripts/main.js

```diff
@@ -1 +1,68 @@
 'use strict';
+
+const createMessage = (text, type) => {
+  const newMessage = document.createElement('div');
+
+  newMessage.dataset.qa = 'notification';
+
+  newMessage.className = `message ${type}`;
+
+  newMessage.textContent = text;
+
+  document.body.append(newMessage);
+};
+
+const firstPromise = new Promise((resolve, reject) => {
+  let click = false;
+
+  document.addEventListener('click', e => {
+    click = true;
+
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    if (click === false) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
```

Comment:
you can simplify it and don't add `click` variable
```suggestion
  document.addEventListener('click', e => {
    resolve('First promise was resolved');
  });

  setTimeout(() => {
    reject(new Error('First promise was rejected'));
  }, 3000);
```